build.proj

```diff
@@ -71,6 +71,18 @@
   <Import Project="dir.targets" />
   <Import Project="dir.traversal.targets" />
   <Import Project="$(ToolsDir)clean.targets" />
+  
+  <Target Name="CollectTemplateFiles" BeforeTargets="UpdateDependencies" >
+    <ItemGroup>
+      <ProjectJsonTemplateFiles Include="$(MSBuildThisFileDirectory)/tools-local/**/project.json.template" />
+      <ProjectJsonTemplateFiles Include="$(MSBuildThisFileDirectory)/src/**/project.json.template" />
+      <ProjectJsonFiles Remove="*.json" />
```

Comment:
Another option, is to use `exclude` if you want to prevent some items from being included during static item group generation